tools/docker/keycloak/.env

```
@@ -0,0 +1,66 @@
+# ---------------------------------------------------------------------------
+# Keycloak
+# ---------------------------------------------------------------------------
+KEYCLOAK_VERSION=26.1
+KEYCLOAK_HTTP_PORT=8042
+# URL through which BOTH the browser and the CAS server reach Keycloak.
+# It is embedded in the OIDC discovery document and in the SAML metadata, so it
+# must be identical on both sides.
+KEYCLOAK_BASE_URL=http://localhost:8042
+KEYCLOAK_REALM=vitamui
+KEYCLOAK_ADMIN_USER=admin
+KEYCLOAK_ADMIN_PASSWORD=changeme
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KEYCLOAK_ADMIN_PASSWORD key should go before the KEYCLOAK_ADMIN_USER key

(UnorderedKey)